File: js/stockPicker.js
exports.picker = function (prices) {
    let result = []
    let maxProfit=0
    for (i = 0; i < prices.length; i++) {
        for (k = i+1; k < prices.length; k++) {
            if ((prices[k] - prices[i]) > maxProfit && i<k) {
                maxProfit=prices[k] - prices[i]
                result= [i,k]
    }}}
    return result
}

File: python/stock_picker.py
def picker(prices):
    max_profit=0
    for i in range (0,len(prices)):
        for k in range (1,len(prices)):
            if prices[k] - prices[i] > max_profit and i<k:
                max_profit=prices[k] - prices[i]
                result= [i,k]
    return result
